src/mainboard/sunxi/nezha/main/Cargo.toml

```diff
@@ -17,8 +17,17 @@ vcell = "0.1.3"
 
 log = { path = "../../../../lib/log" }
 oreboot-arch = { path = "../../../../arch", features = ["riscv64"] }
-oreboot-soc = { path = "../../../../soc", features = ["sunxi_d1"] }
-oreboot_compression = { path = "../compression" }
+
+[dependencies.oreboot_compression]
+path = "../../../../lib/compression"
+
+[dependencies.oreboot-soc]
+path = "../../../../soc"
+features = ["sunxi_d1"]
+
+[dependencies.riscv]
```

Comment:
this is already here further up - the dependencies are divided in 2 blocks: first external crates, then local (oreboot's own) crates

suggestion: let's keep the inline style; the multiline style seems a bit confusing, resembling different sections

Reply:
oh I thought that was your stuff